lib/util/default-container.js

```diff
@@ -23,7 +23,7 @@ const { mailer } = require('../external/mail');
 const mail = mailer(mergeRight(config.get('default.email'), { env }));
 
 // get a sentry and configure errors.
-const { Sentry } = require('../external/sentry');
+const Sentry = require('../external/sentry').getInstance();
```

Comment:
I feel vaguely uncomfortable with `getInstance()` being called here. It sets up a run-order requirement that didn't exist before whereby this file has to be `require()`d after another file that has already created a Sentry instance. Could we get rid of `require('../external/sentry.js')` altogether in run-server.js, then move the Sentry `init()` here? This file already calls a bunch of other `init()` functions, so I feel like it'd be a consistent pattern.

Reply:
We could, but then any problems in `run-server.js` will not be captured by Sentry.

Reply:
Sentry docs recommend:

> [!WARNING]
> Import ... before any other modules to ensure Sentry initializes early. If you initialize later, auto-instrumentation and modules like database monitoring, agent monitoring, tracing may not work.

https://docs.sentry.io/platforms/javascript/guides/node/#apply-instrumentation-to-your-app

Reply:
> Someone could add something to the top of run-server.js without realizing the consequences.

Wherever we init Sentry, maybe we should add a comment somewhere saying that Sentry needs to be set up first.